internal/operands/vm-delete-protection/reconcile.go

```diff
@@ -0,0 +1,82 @@
+package vm_delete_protection
+
+import (
+	admissionregistrationv1 "k8s.io/api/admissionregistration/v1"
+	utilruntime "k8s.io/apimachinery/pkg/util/runtime"
+	"kubevirt.io/ssp-operator/internal/common"
+	"kubevirt.io/ssp-operator/internal/operands"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+// Define RBAC rules needed by this operand:
+// +kubebuilder:rbac:groups=admissionregistration.k8s.io,resources=validatingadmissionpolicies,verbs=get;list;create;watch;update
+// +kubebuilder:rbac:groups=admissionregistration.k8s.io,resources=validatingadmissionpolicybindings,verbs=get;list;create;watch;update
+
+const (
+	operandName                                     = "vm-delete-protection"
+	operandComponent                                = common.AppComponetVMDeletionProtection
+	virtualMachineDeleteProtectionPolicyName        = "ssp-vm-deletion-protection-policy"
+	virtualMachineDeleteProtectionPolicyBindingName = "ssp-vm-deletion-protection-binding"
```

Comment:
Maybe we can use fully qualified names for the cluster resources, like `ssp.kubevirt.io/vm-deletion-protection` ?

Can you please remove the suffixes `-policy` and `binding`? It is redundant, because we know the object type.

Reply:
I've used `ssp.kubevirt.io-vm-delete-protection`, because `/` symbol is not allowed in object names. However, I'm not convinced about it, it sounds weird imho.

Reply:
Dropped suffixes. Without them, both names are the same, so I've also combined the strings into a single one.

Reply:
I understand. What about `kubevirt-vm-delete-protection` or `ssp-kubevirt-vm-delete-protection`?

Reply:
IMHO the first one is better.

Reply:
Adjusted.